Plugins/Public/base_plugin/Main.cpp

```diff
@@ -1114,6 +1117,14 @@ BOOL WINAPI DllMain(HINSTANCE hinstDLL, DWORD fdwReason, LPVOID lpvReserved)
 			delete base->second;
 		}
 
+		for (uint customSolar : customSolarList)
+		{
+			int isAlive = pub::SpaceObj::ExistsAndAlive(customSolar);
+			if(isAlive)
```

Comment:
The declaration of the `isAlive` variable doesn't add that much value in my opinion.
```suggestion
			if (pub::SpaceObj::ExistsAndAlive(customSolar))
```